cristin-project/src/test/java/no/unit/nva/cristin/common/client/AuthenticatedApiClientTest.java

```diff
@@ -0,0 +1,34 @@
+package no.unit.nva.cristin.common.client;
+
+import nva.commons.core.ioutils.IoUtils;
+import nva.commons.core.paths.UriWrapper;
+import org.junit.jupiter.api.BeforeEach;
+
+import java.net.URI;
+import java.net.http.HttpResponse;
+import java.nio.file.Path;
+
+import static java.net.HttpURLConnection.HTTP_CREATED;
+import static no.unit.nva.cristin.model.Constants.CRISTIN_API_URL;
+import static no.unit.nva.cristin.model.Constants.PROJECTS_PATH;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class AuthenticatedApiClientTest {
+
+    private ApiClient cristinApiClient;
+
+    @BeforeEach
+    void setUp() {
+        cristinApiClient = new ApiClient(CristinAuthenticator.getHttpClient());
+    }
+
+    //    @Test
```

Comment:
Since we're not testing, is the test needed?

Reply:
Removed test, added simple javadoc instead